Documentation/botr/readytorun-format.md

```diff
@@ -70,7 +68,7 @@ ReadyToRun images from other CLI images with ManagedNativeHeader (e.g. NGen imag
 
 ### READYTORUN_HEADER::MajorVersion/MinorVersion
 
-The current format version is 1.1. MajorVersion increments are meant for file format breaking changes. 
+The current format version is 3.0. MajorVersion increments are meant for file format breaking changes. 
```

Comment:
```suggestion
The current format version is 3.1. MajorVersion increments are meant for file format breaking changes. 
```

The tables below talk about 3.1

Reply:
Thanks for pointing that out, fixed in 8th commit.